packages/webpack-plugin/src/webpack-config-stylable-excludes.ts

```diff
@@ -54,3 +54,31 @@ function safelyWalkJSON(
         }
     }
 }
+
+export function bundleLibs(config: Configuration, packages: string[], isServer: boolean) {
+    if (isServer) {
+        if (Array.isArray(config.externals) && config.externals.length === 1) {
```

Comment:
Is the `externals` field in nextjs always a function config? can something change it before the user has a chance to call our modifier function?

Reply:
I don't think that's the case, but we'll encounter it as it happens.